tests/src/test/java/alluxio/master/file/ConcurrentFileSystemMasterCreateTest.java

```diff
@@ -87,8 +86,10 @@ public static void afterClass() throws Exception {
   }
 
   @Before
-  public void before() {
+  public void before() throws Exception {
     mFileSystem = FileSystem.Factory.get();
+    // Mount the sleeping UFS, to always the sleeping UFS over the ufs cluster.
```

Comment:
nit: I think this comment is inaccurate - it's still possible to use the default UFS by accessing a path not under `/sleep/`